applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -294,29 +307,25 @@ See :ref:`app_build_system`: for more information on the |NCS| configuration sys
 Native TLS
 ----------
 
-By default, the secure socket (TLS/DTLS) is offloaded onto the modem.
-However, if you need customized TLS/DTLS features that are not supported by the modem firmware, you can use native TLS instead.
+By default, the secure socket (TLS) is offloaded onto the modem.
+If you need customized TLS features that are not supported by the modem firmware, you can use native TLS instead.
+Native TLS uses the Mbed TLS library in Zephyr to establish secure connectivity.
 Currently, the SLM application can be built to use native TLS for the following services:
 
 * Secure socket
+* TLS Proxy client
 * TLS Proxy server
 * HTTPS client
 
-If native TLS is enabled, you must use the ``AT#XCMNG`` command to store the credentials.
-
-.. note::
-
-   The modem needs to be in an offline state when storing the credentials.
-   The SLM application supports security tags ranging from ``0`` to ``214748364``.
+With native TLS, the credentials are stored in the Zephyr settings storage with the ``AT#XCMNG`` command.
 
 The configuration options that are required to enable the native TLS socket are defined in the :file:`overlay-native_tls.conf` file.
 
 .. note::
 
-   Native TLS sockets have the following limitations:
+   Native TLS services have the following limitations:
 
    * PSK, PSK identity, and PSK public key are currently not supported.
```

Comment:
Though they (PSK, PSK identity) are defined in the AT command and the code. Is this planned to be fixed/supported?

Reply:
Good catch. It is supported now.

Reply:
This still needs to be fixed.

Reply:
What about the type `5 – Public key (ASCII text)`? It is not defined in the header nor the documentation, so it seems like it should be added here.

Reply:
It is for completely different use case than native TLS. I think that it has been mistake to have it documentation to begin with.
 – Public key (ASCII text). Used in authenticated AT commands.